### ISSUE TITLE
Write() takes exactly X arguments

### ISSUE THREAD

**Issue by figbux** [l29ah/libmpsse#9]
Hello,
Installed mpsse in normal way and trying to run spiflash.py now. I get this error:
```
$ python2 spiflash.py -i
FT2232 Future Technology Devices International, Ltd initialized at 15000000 hertz
Traceback (most recent call last):
  File "spiflash.py", line 233, in <module>
    main()
  File "spiflash.py", line 204, in main
    for byte in spi.ChipID():
  File "spiflash.py", line 81, in ChipID
    self.flash.Write(self.IDCMD)
  File "/usr/lib/python2.7/site-packages/mpsse.py", line 250, in Write
    if _mpsse.Write(self.context, data) == MPSSE_FAIL:
TypeError: Write() takes exactly 3 arguments (2 given)
```
Weird things is, if I call `Write()` with a random "3rd" argument, it now says `TypeError: Write() takes exactly 2 arguments (3 given)`.

What may be the cause of this issue?

Thanks.

**Comment by figbux** [comment]
Reverting https://github.com/l29ah/libmpsse/commit/148b0702c30f8ed29ddd962bb920185d36bb9a48 does not fix.

**Comment by elimohl** [comment]
@figbux can you please check if it works as intended with this fix: https://github.com/elimohl/libmpsse/commit/58757d7972849276b615ad5737e357ac2a36d3e4 ?

**Comment by figbux** [comment]
Hey,
Thank you! It's working. Can you please make a PR to this repo; there are so many libmpsse forks out there and very hard to follow which one is the most stable, especially for newcomers :)
